tests/sandbox/test_get_codesystem.py

```diff
@@ -14,7 +14,27 @@
 class TestGetCodesystem(SandboxTest):
     @pytest.fixture
     def unauthorised_actors(self) -> Iterable[Actor]:
-        return []
+        return self.unauthorised_actors_list()
+
+    def unauthorised_actors_list(self) -> List[Actor]:
```

Comment:
Could the result of this method be moved into the existing method above?

Reply:
pytest gave me an error for calling the fixture directly. One of their suggested workarounds was to have the fixture call a method